src/campaigns.ts

```diff
@@ -446,7 +441,7 @@ export const campaigns: Campaign[] = [
             endTimestampExclusive,
             rewardAmount: '25000',
             excludedReferrers: {},
-            previousResultDirectories: [],
+            previousKpiFiles: [''],
```

Comment:
should this be just an empty array? i _think_ `readKpiFile` expects non empty strings, but maybe you have plans for this in a follow up